Gateway/Http/Client/TransactionPosCloudSync.php

```diff
@@ -30,36 +30,41 @@
 
 class TransactionPosCloudSync implements ClientInterface
 {
+    /**
+     * @var int
+     */
+    protected $storeId;
 
     public function __construct(
         \Magento\Framework\Model\Context $context,
         \Magento\Framework\Encryption\EncryptorInterface $encryptor,
         \Adyen\Payment\Helper\Data $adyenHelper,
         \Adyen\Payment\Logger\AdyenLogger $adyenLogger,
         \Adyen\Payment\Model\RecurringType $recurringType,
+        \Magento\Store\Model\StoreManagerInterface $storeManager,
         array $data = []
     ) {
         $this->_encryptor = $encryptor;
         $this->_adyenHelper = $adyenHelper;
         $this->_adyenLogger = $adyenLogger;
         $this->_recurringType = $recurringType;
         $this->_appState = $context->getAppState();
+        $this->storeManager = $storeManager;
```

Comment:
please create a property for the storemanager also

Reply:
do we need to expose the storeManager. As it is only used for storeId we can just do: 
```
this->storeId = $storeManager->getStore()->getId();
```